sample_global_config.ini

```diff
@@ -7,5 +7,13 @@ DIRECTORY = /tmp
 # logging level
 LOG_LEVEL = INFO
 
+# port to listen for requests on
+PORT = 8080
+
+# neighbor servers to get data from
+SERVERS = http://example.org/bct-server1, http://example.org/bct-server2
+
+# how often to sync from neighbors
```

Comment:
In what - seconds, milliseconds, minutes ?

Reply:
seconds.  I added that doc string